src/rates.ts

```diff
@@ -50,7 +50,7 @@ import {
 
 const { bridgeCurrencies, preferredCryptoFiatPairs } = config
 
-const PRECISION = 20
```

Comment:
Should update at least AssetMap per our new housekeeping convention: 
`Set<string>, Map<string, boolean> // Use for dynamic data`